action.yml

```diff
@@ -24,6 +24,12 @@ inputs:
     description: 'Optional Github token for fetching a release. Only specify when using custom toolchains from non-public Github releases.'
     required: false
 
+  # common
+  installer-args:
```

Comment:
@compnerd Do we care to support this on macOS? It looks like there's an installer-like executable there too

Reply:
Yes, we should support installation on macOS as well. I don’t think that the installer has arguments there though.

Reply:
That's what I meant, then I don't need more code in this PR.